### PR TITLE
Add amiga robot to robots.ros.org

### PR DESCRIPTION
**Pull request by jerrycowell96**

### CODE REVIEW

**Comment by jerrycowell96** [comment]
@tfoote can you please review and merge this branch?

File: _posts/2023-05-26-farm-ng-amiga.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- 'precision farming'
+- 'research'
+- 'robot'
+- 'robotics'
+- 'ground'
+- 'mobile-robot'
+- 'outdoor'
+twitter_text: Checkout the Farm-ng Amiga
+website: https://farm-ng.com/pages/for-research
+wiki_homepage: https://farm-ng.com/pages/for-research
```
Comment (tfoote) [inline _posts/2023-05-26-farm-ng-amiga.md]: This doesn't appear to have any ROS API documentation. Please use the ROS interface link below in the description.

File: _posts/2023-05-26-farm-ng-amiga.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Farm-ng Amiga
+date: 2023-05-26
+description:
+icon: /assets/img/robots/farm-ng-amiga/amiga.png
```
Comment (tfoote) [inline _posts/2023-05-26-farm-ng-amiga.md]: This is the same as the full size image. It should be smaller following that icon sizing guide for the tile layout.

**Review by tfoote — CHANGES_REQUESTED** [review]
Thanks for submitting this. I have two pieces of feedback. With those resolved it looks good.

**Comment by jerrycowell96** [comment]
@tfoote Thank you for your review. You will find these changes there and the conflicts resolved.

File: _posts/2023-05-26-farm-ng-amiga.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- 'precision farming'
+- 'research'
+- 'robot'
+- 'robotics'
+- 'ground'
+- 'mobile-robot'
+- 'outdoor'
+twitter_text: Checkout the Farm-ng Amiga
+website: https://farm-ng.com/pages/for-research
+wiki_homepage: http://wiki.ros.org/Robots/farm-ng-amiga
```
Comment (tfoote) [inline _posts/2023-05-26-farm-ng-amiga.md]: The link is now following our usual pattern, but there doesn't appear to be any content here: http://wiki.ros.org/Robots/farm-ng-amiga which should tell developers about the ROS API.

File: _posts/2023-05-26-farm-ng-amiga.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Farm-ng Amiga
+date: 2023-05-26
+description:
+icon: /assets/img/robots/farm-ng-amiga/farm-ng-logo.png
```
Comment (tfoote) [inline _posts/2023-05-26-farm-ng-amiga.md]: This icon should be 80x80 picture of the robot which is what people will see in the list of the robots.

**Review by tfoote — CHANGES_REQUESTED** [review]
Thanks for iterating. That's good progress, but neither really meet our needs for contributing. 

Please read through: https://robots.ros.org/contributing/ for more details.

**Comment by jerrycowell96** [comment]
Oh my apologies. I did not understand at first but I'll make those changes immediately. Thank you.

**Comment by jerrycowell96** [comment]
@tfoote Hi, I have resolved the above conflicts. Can you please review this PR and approve. If you have any other suggestions or question please do not hesitate to let me know.

File: test_site.bash [hunk 5]
```diff
@@ -3,4 +3,5 @@ set -x
 
 set -o errexit
 
-ghrocker --config _config.yml localtest_config.yml -- .
\ No newline at end of file
+#ghrocker --config _config.yml localtest_config.yml -- .
+ghrocker --config _config.yml -- .
```
Comment (tfoote) [inline test_site.bash]: This looks like an unrelated change that got pulled in.

**Review by tfoote — CHANGES_REQUESTED** [review]
One small change. It looks like you tweaked the test script while working and included it in the PR. 

Thanks for filling out the wiki page.

File: test_site.bash (hunk 5 above)
Comment (jerrycowell96) [reply test_site.bash]: Yikes okay i'll change that right now.

Comment (jerrycowell96) [reply test_site.bash]: Should be reverted back to original. Let me know if anything else comes up.

**Review by tfoote — APPROVED** [review]

**Comment by tfoote** [comment]
Thanks for the iteration. it should go live shortly.

**Pull request merged**